apps/web/src/pages/activities/hooks/useNotificationStatus.ts

```diff
@@ -13,7 +13,7 @@ export const useNotificationStatus = (notification: any): JobStatusEnum => {
         if (prev === JobStatusEnum.FAILED) {
           return prev;
         }
-        if ([JobStatusEnum.PENDING, JobStatusEnum.DELAYED].includes(prev)) {
+        if ([JobStatusEnum.PENDING, JobStatusEnum.QUEUED].includes(prev)) {
```

Comment:
Yes this may include regular jobs sometimes briefly,  was just in two minds.(one way is it shows all queued jobs at that moment so we can see all jobs, other way is why to show regular queued jobs even for short times), if needed we can add delay > 0 condition in the final PR